plugins/hypervisors/kvm/src/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPvlanSetupCommandWrapper.java

```diff
@@ -58,11 +59,11 @@ public Answer execute(final PvlanSetupCommand command, final LibvirtComputingRes
         String result = null;
         try {
             final String guestBridgeName = libvirtComputingResource.getGuestBridgeName();
-            final int timeout = libvirtComputingResource.getTimeout();
+            final Duration timeout = libvirtComputingResource.getTimeout();
 
             if (command.getType() == PvlanSetupCommand.Type.DHCP) {
                 final String ovsPvlanDhcpHostPath = libvirtComputingResource.getOvsPvlanDhcpHostPath();
-                final Script script = new Script(ovsPvlanDhcpHostPath, timeout, s_logger);
+                final Script script = new Script(ovsPvlanDhcpHostPath, timeout.getMillis(), s_logger);
```

Comment:
Scripts is an external package, hope they will upgrade to Duration after Java 1.8.